### ISSUE TITLE
[MU4 Issue] Playback unevenness

### ISSUE THREAD

**Issue by bkunda** [musescore/MuseScore#13352]
**Describe the bug**
Playback of certain note values is uneven at certain tempi.
This issue affects our perception of the performed rhythm. It seems to vary depending on the instrument, as well as the shortness of the note relative to the tempo.

Please see this video for a fairly comprehensive demonstration, which also shows how this issue can be reproduced (9mins): https://www.dropbox.com/s/hhnnxothnuuro80/note-unevenness.mov?dl=0

**Expected behavior**
Playback rhythm should be as even as possible (see comparison examples in the above video from MuseScore 3.6 and NotePerformer in Sibelius).

**Platform information**
 - MacOS 12.3.1 (M1)

**Additional context**
MuseScore test file I used in the video: [note-unevenness-files.zip](https://github.com/musescore/MuseScore/files/9566275/note-unevenness-files.zip)

This issue may be a duplicate of #12870, although on my hearing I suspect this problem has more to do with note duration/playback length than the actual tempo at which the music is being played back (although tempo certainly affects it).

Some further useful examples are provided in Discord by @MarcSabatella here: https://discord.com/channels/818804595450445834/939880479887327302/1019359064276336650

**Comment by Tantacrul** [comment]
Just going to link to @MarcSabatella's issue here, which may help to give more examples: https://github.com/musescore/MuseScore/issues/12870#issuecomment-1245527331

**Comment by Tantacrul** [comment]
#https://github.com/musescore/MuseScore/issues/12870#issuecomment-1245527331

**Comment by HemantAntony** [comment]
This his discussion: #13337

**Comment by MarcSabatella** [comment]
Thanks for posting this!  FWIW, what I hear with the repeated notes in the first example of your video sounds more like a *volume* difference (loud-soft-loud-soft) than duration or rhythm.  And I think I hear that as well in my examples.

**Comment by bkunda** [comment]
Re-tested on MacOS and the unevenness is no longer so discernible. I think this is pretty much fixed!

(FWIW, I did still detect some slight unevenness with the flute playback, but I think I've narrowed this down to a difference in that particular instrument's timbral properties from C#6 and upwards, which causes discernible "bumps" when moving between the flute's second and third octaves. This seems to be an issue with the MS Basic flute sound though, and I could not hear the same thing using other flute VSTs, so I would therefore class this as a separate, and for now less critical, issue).
